files/config/rsyslog.default

```diff
@@ -3,7 +3,7 @@ if $syslogfacility-text == 'kern' and (($msg startswith ' IPT:') or ($msg starts
   stop
 }
 
-if $syslogfacility-text == 'kern' and (($msg startswith ' filter_IN_99_simp_DROP:') or ($msg startswith 'filter_IN_99_simp_DROP:')) then {
+if $syslogfacility-text == 'kern' and (($msg startswith ' IN_99_simp_DROP:') or ($msg startswith 'IN_99_simp_DROP:') or ($msg startswith ' filter_IN_99_simp_DROP:') or ($msg startswith 'filter_IN_99_simp_DROP:')) then {
```

Comment:
Why are both IN_xxx and filter_IN_xxx required?

Reply:
I'm removing the `filter_` rules. Those were from the experimental version of our `firewalld` management but nobody should be using that set of modules since there ended up being bugs.